### DISCUSSION TITLE
What is the best way to handle Evaluation exception?

### DISCUSSION THREAD

**Discussion by xmxnt** [evrete/evrete#38] [Q&A]
Suppose if I have 50 rules in knowledge and one of them is misconfigured(or have incorrect MVEL syntax, incorrect datatype at runtime). I want the rule engine to ignore this rule and continue executing the rest. Is it possible?

**Comment by andbi** [comment]
Hi,

this is a planned feature for the next release which is due next week. I'm turning this discussion into a feature request so you could track it and get notified.

**Reply by xmxnt** [reply]
That is great! Thanks.

**Reply by andbi** [reply]
Done. The official docs for 3.0.01 haven't  been published yet, but the the handler and its use are pretty well documented:

https://github.com/evrete/evrete/blob/master/evrete-core/src/main/java/org/evrete/api/RuleBuilderExceptionHandler.java
